src/NavItem/index.jsx

```diff
@@ -95,7 +95,7 @@ NavItem.propTypes =
     /*
     *  Item represents the current page and/or section
      */
-    isCurrent     : PropTypes.bool,
+    isCurrentPage : PropTypes.bool,
```

Comment:
This one was specified as prop, but was missing in props validation (there's also props validation for `isCurrent` )